src/ProjectTemplates/Web.ProjectTemplates/Worker-CSharp.csproj.in

```diff
@@ -1,8 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Worker">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <OutputType>Exe</OutputType>
+    <UserSecretsId>dotnet-Company.Application1-53bc9b9d-9d6a-45d4-8429-2a2761773502</UserSecretsId>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
```

Comment:
Should this be set in the worker SDK? Like presumably we do in the web SDK?

Reply:
Glenn and Damian asked for it to be surfaced in the csproj where it's more discoverable. It defaults to true in the SDK.

Reply:
So it's `true` in two places? That sounds odd to me but it doesn't bother me too much... I feel that if someone truly cared about it, having this setting in their face wouldn't help them one way or another. And for people who don't care about it, it also wouldn't help.

Reply:
Why is server GC even on for workers? I also agree with Eilon, I dunno why it's ehre.